ui/src/main/java/org/pentaho/di/ui/core/database/wizard/CreateDatabaseWizardPage1.java

```diff
@@ -45,11 +45,14 @@
 import org.pentaho.di.core.Const;
 import org.pentaho.di.core.database.DatabaseMeta;
 import org.pentaho.di.core.database.GenericDatabaseMeta;
+import org.pentaho.di.core.exception.KettleException;
```

Comment:
need to update copyright on all modified files in pentaho-kettle.

Reply:
updated